src/_06_Intro_To_Hash_Maps/_02_LogSearch.java

```diff
@@ -1,33 +1,80 @@
 package _06_Intro_To_Hash_Maps;
 
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+import java.util.HashMap;
 
-public class _02_LogSearch {
-  /* 
-	 * Crate a HashMap of Integers for the keys and Strings for the values.
-	 * Create a GUI with three buttons. 
-	 * Button 1: Add Entry
-	 * 				When this button is clicked, use an input dialog to ask the user to enter an ID number.
-	 * 				After an ID is entered, use another input dialog to ask the user to enter a name.
-	 * 				Add this information as a new entry to your HashMap.
-	 * 
-	 * Button 2: Search by ID
-	 * 				When this button is clicked, use an input dialog to ask the user to enter an ID number.
-	 * 				If that ID exists, display that name to the user.
-	 * 				Otherwise, tell the user that that entry does not exist.
-	 * 
-	 * Button 3: View List
-	 * 				When this button is clicked, display the entire list in a message dialog in the following format:
-	 * 				ID: 123  Name: Harry Howard
-	 * 				ID: 245  Name: Polly Powers
-	 * 				ID: 433  Name: Oliver Ortega
-	 * 				etc...
-	 * 
-	 * When this is complete, add a fourth button to your window.
-	 * Button 4: Remove Entry
-	 * 				When this button is clicked, prompt the user to enter an ID using an input dialog.
-	 * 				If this ID exists in the HashMap, remove it. Otherwise, notify the user that the ID
-	 * 				is not in the list. 
-	 *
-	 * */
-	
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JOptionPane;
+import javax.swing.JPanel;
+
+public class _02_LogSearch implements ActionListener {
+	HashMap<Integer, String> map = new HashMap<Integer, String>();
```

Comment:
I don't think that you need to put anything in the diamond operator when you instantiate/initialize a HashMap 🍶